server/service/src/sync/test/test_data/requisition.rs

```diff
@@ -39,7 +39,14 @@ const REQUISITION_REQUEST: (&'static str, &'static str) = (
       "programID": "F36DBBC6DBCA4528BDA2403CE07CB44F",
       "lastModifiedAt": 1594273006,
       "is_emergency": false,
-      "isRemoteOrder": false
+      "isRemoteOrder": false,
+      "om_created_datetime": "",
+      "om_sent_datetime": "",
+      "om_finalised_datetime": "",
+      "om_expected_delivery_date": "0000-00-00", 
```

Comment:
@andreievg @clemens-msupply in mSupply this is a `Date` field type rather than string. Might be an idea to go to string if strictly for omSupply?